packages/angular_devkit/build_angular/src/utils/i18n-inlining.ts

```diff
@@ -40,7 +40,7 @@ function emittedFilesToInlineOptions(
       es5,
       outputPath,
       missingTranslation,
-      setLocale: emittedFile.name === 'main',
+      setLocale: emittedFile.name === 'main' || emittedFile.name === 'vendor',
```

Comment:
Is it possible for the locale to need to be set in lazy chunks?

Reply:
From the point of view of Angular, we only need to set it in `main.js` so that we can pick it up in Angular services and pipes. And I guess, since it is global, that once it is set it will automatically be available to all lazy loaded chunks.

Reply:
Then what's the situation where it needs to be set in vendor?